docs_src/microservices/core/command/Ch-Command.md

```diff
@@ -367,10 +367,10 @@ Example of making get command request via messaging:
   }
 }
 ```
-2. Receive command response message from external MQTT broker on topic `edgex/commandquery/response/#`:
+2. Receive command response message from external MQTT broker on topic `edgex/command/response/#`:
 ```json
 {
-  "ReceivedTopic":"edgex/device/command/response/device-virtual/Random-Boolean-Device/Bool/get",
+  "ReceivedTopic":"edgex/response/device-virtual/e6e8a2f4-eb14-4649-9e2b-175247911369",
```

Comment:
The ReceivedTopic, aka Response topic, on external MQTT will be `edgex/command/response/Random-Boolean-Device/Bool/get`

See https://github.com/edgexfoundry/edgex-go/blob/main/internal/core/command/controller/messaging/external.go#L129